napari/utils/transforms/transforms.py

```diff
@@ -93,6 +95,19 @@ def expand_dims(self, axes: Sequence[int]) -> 'Transform':
             trans._('Cannot subset arbitrary transforms.', deferred=True)
         )
 
+    @cached_property
```

Comment:
```suggestion
    @property
```

Nit: this always returns false, so just a plain property seems fine.